enterprise-ws-spec/src/main/asciidoc/ServerProgrammingModel.adoc

```diff
@@ -351,295 +344,180 @@ web.xml or a partial web.xml:
+jakarta.xml.ws.WebServiceProvider annotated Service Implementation Beans
 can be run either as a Stateless or Singleton Session EJB in an EJB
-container or as a JAX-WS service endpoint in a web container. The two
-programming models are fully defined in sections
+container or as a Jakarta XML Web Services service endpoint in a web container. 
+The two programming models are fully defined in sections
 link:#anchor-43[5.3.2.3] and link:#anchor-44[5.3.2.4].
 
 [#anchor-43]
 ===== EJB container programming model
 
-A Stateless Session Bean, as defined by the Enterprise JavaBeans
-specification, can be used to implement a Web service to be deployed in
-the EJB container. A Singleton Session Bean, as defined by the EJB 3.1
-specification, can be used to implement a JAX-WS Web service to be
-deployed in the EJB container.
+A Stateless Session Bean, as defined by the Jakarta Enterprise Beans 4.0 specification, 
+can be used to implement a Web service to be deployed in the EJB container. 
+A Singleton Session Bean, as defined by the Jakarta Enterprise Beans 4.0 specification, 
+can be used to implement a Jakarta XML Web Services Web service to be deployed 
+in the EJB container.
 
 A Stateless Session Bean does not have to worry about multi-threaded
 access. The EJB container is required to serialize request flow through
 any particular instance of a Service Implementation Bean. A Singleton
 Session Bean is intended to be shared and supports concurrent access.
-The access rules are specified in the 4.8.5 section of EJB 3.1.
+The access rules are specified in the Jakarta Enterprise Beans 4.0 specification.
 
 The requirements for creating a Service Implementation Bean as a
 Stateless or Singleton Session EJB are repeated in part here.
 
-* With JAX-WS, the Service Implementation Bean class must be annotated
-with either javax.jws.WebService or javax.xml.ws.WebServiceProvider
-annotation. See section 5.3.2.1 and 5.3.2.2
-* For developers starting from Java using JAX-WS, javax.jws.WebService
-annotation on Service Implementation Bean may optionally reference an
-SEI but is not required to do so. If SEI is not specified, the Service
-Implementation Bean class implicitly defines a SEI as required by
-section 3.3 of JAX-WS specification. The Service Implementation Bean
-methods are not required to throw javax.rmi.RemoteException. The
+* The Service Implementation Bean class must be annotated with either 
+jakarta.jws.WebService or jakarta.xml.ws.WebServiceProvider annotation. 
+See section 5.3.2.1 and 5.3.2.2
+* For developers starting from Java, jakarta.jws.WebService annotation 
+on Service Implementation Bean may optionally reference an SEI but is 
+not required to do so. If SEI is not specified, the Service Implementation 
+Bean class implicitly defines a SEI as required by section 3.3 of 
+Jakarta XML Web Services specification. The Service Implementation Bean
+methods are not required to throw jakarta.rmi.RemoteException. The
```

Comment:
hmm, `rmi` is definitely removed from SE 11 but I do not think it is going to be moved to jakarta package. Looking into [the plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan), Support for Distributed Interoperability in the EJB 3.2 Core Specification, Chapter 10 is pruned from EE 9, so `javax.rmi.RemoteException` should remain as is

Reply:
ok, will keep javax.rmi.RemoteException.